internal/cli/opsmanager/clusters/clusters.go

```diff
@@ -39,6 +39,7 @@ func Builder() *cobra.Command {
 		IndexesBuilder(),
 		UnmanageBuilder(),
 		RestartBuilder(),
+		ReclaimFreeSpaceBuilder(),
```

Comment:
[nit] just spotted this but can we move restart before Shutdown and ReclaimFreeSpaceBuilderafter StartupBuilder, seems to make sense (to me) to keep all this modification actions together, I would even move apply before create but that is a bit more fuzzy in my head